### PR TITLE
Initial training structure

### PR DESCRIPTION
**Pull request by AndreLaranjeira**
* Altered dataset loading to use 'keras.datasets'.
* Formatted data based on backend, input range and output range.
* Made changes to the CNN model to account for the data formatting.
* Implemented basic structure for fitting the model to the data.

### CODE REVIEW

**Review by labm1997 — APPROVED** [review]
Very good training I've gotten a great (0.9745) accuracy on validation pretty fast
